users/vieting/experiments/librispeech/librispeech_960_pretraining/wav2vec2/config_02_fairseq_phoneme.py

```diff
@@ -181,11 +209,13 @@ def run_fairseq_pretraining_positive_sampling(num_positives: int = 10):
     job = FairseqHydraTrainingJob(fairseq_config, **fairseq_training_args)
     job.add_alias(os.path.join(prefix_name, exp_name, "pretraining"))
     tk.register_output(f"{prefix_name}/{exp_name}/pretraining/scores.png", job.out_plot_se)
-    return job    
+    return job
+
```

Comment:
Was this on purpose?

Reply:
No, I am actually not quite sure where this even comes from. Can we leave it like that or should I try to revert that?

Reply:
I'll try to fix it here.

Reply:
Ok that doesn't seem to work. I think this occurred because there was whitespace after `job` and you (or your IDE) removed it. I'm fine with removing the whitespace but the additional new line should be removed. Could you please fix this in your branch and push it?